tutorials/how-to-implement-rag-generativeapis/index.mdx

```diff
@@ -34,28 +34,35 @@ In this tutorial, you will learn how to implement RAG using LangChain, a leading
 
 ### Install required packages
 
-Run the following command to install the required packages:
+1. Run the following command to install the required Python packages:
```

Comment:
This section is now the only one in the whole tutorial having numbered steps 1. and 2.

Reply:
@tgenaitay @fpagny we can remove the steps numbers, as we have less than 3 in the tutorial.
Sorry, I ended up forgetting to remove them yesterday after I finished the review. 🤓